### PR TITLE
[native] Add support for DEPENDENCY_DIR, INSTALL_PREFIX, PYTHON_VENV

### PR DESCRIPTION
**Pull request by majetideepak**
## Description

DEPENDENCY_DIR and INSTALL_PREFIX help developers manage dependencies.
PYTHON_VENV is used to configure the clang-format package.

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/README.md
Comment (steveburnett) [inline presto-native-execution/README.md:35]:
```suggestion
location (for example, `/usr/local`) on linux.
```

File: presto-native-execution/README.md
Comment (steveburnett) [inline presto-native-execution/README.md:36]:
```suggestion
Using the default install location `/usr/local` on macOS is discouraged because this
```

File: presto-native-execution/README.md
Comment (steveburnett) [inline presto-native-execution/README.md:39]:
```suggestion
Manually add the `INSTALL_PREFIX` value in the IDE or bash environment so subsequent Velox builds can use the installed packages.
```
Suggest revising this to describe in one sentence what you want the user to do, then separate out the example as a second sentence. I've tried doing this here, let me know if my suggestion makes sense.

File: presto-native-execution/README.md
Comment (steveburnett) [inline presto-native-execution/README.md:41]:
```suggestion
```

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks for the doc! A couple of minor nits, and a suggestion to rephrase for easier understanding.

**Review by czentgr — APPROVED** [review]
Looks good.

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

File: presto-native-execution/scripts/setup-macos.sh
Comment (majetideepak) [inline presto-native-execution/scripts/setup-macos.sh:22]: @czentgr I added this change as well.

**Review by czentgr — APPROVED** [review]

**Comment by majetideepak** [comment]
@tdcmeehan can you please help approve and merge this? Thanks.
This needs a committer approval due to the changes to .gitignore.

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**
